site/en/docs/web-platform/navigation-api/index.md

```diff
@@ -316,50 +295,50 @@ In fact, it will always be `undefined` in those cases.
 It also includes `back()`, `forward()` and `reload()`.
-These methods are all handled—just like `navigate()`—by the centralized "navigate" event handler.
+These methods are all handled—just like `navigate()`—by the centralized "navigate" event listener.
 
 ### Form Submissions
 
-Secondly, HTML `<form>` submission via POST is a special type of navigation, and the App History API can intercept it.
-While it includes an additional payload, the navigation is still handled centrally by the "navigate" handler.
+Secondly, HTML `<form>` submission via POST is a special type of navigation, and the Navigation API can intercept it.
+While it includes an additional payload, the navigation is still handled centrally by the "navigate" listener.
 
-Form submission can be detected by looking for the `formData` property on the `AppHistoryNavigateEvent`.
+Form submission can be detected by looking for the `formData` property on the `NavigateEvent`.
 Here's an example that simply turns any form submission into one which stays on the current page via `fetch()`:
 
 ```js
-appHistory.addEventListener('navigate', event => {
-  if (event.formData && event.canTransition) {
+navigation.addEventListener('navigate', navigateEvent => {
+  if (navigateEvent.formData && navigateEvent.canTransition) {
     // User submitted a POST form to a same-domain URL
     // (If canTransition is false, the event is just informative:
     // you can't intercept this request, although you could
     // likely still call .preventDefault() to stop it completely).
 
     const submitToServer = async () => {
-      await fetch(event.destination.url, {
+      await fetch(navigateEvent.destination.url, {
         method: 'POST',
-        body: event.formData,
+        body: navigateEvent.formData,
       });
       // You could navigate again with {replace: true} to change the URL here,
       // which might indicate "done"
     };
-    event.transitionWhile(submitToServer());
+    navigateEvent.transitionWhile(submitToServer());
   }
 });
 ```
 
 ## What's missing?
 
-Despite the centralized nature of the "navigate" event handler, the current App History API specification doesn't trigger "navigate" on a page's first load.
+Despite the centralized nature of the "navigate" event listener, the current Navigation API specification doesn't trigger "navigate" on a page's first load.
 And for sites which use [Server Side Rendering][ssr-definition] (SSR) for all states, this might be fine—your server could return the correct initial state, which is the fastest way to get content to your users.
 But sites that leverage client-side code to create their pages may need to create an additional function to initialize their page.
-This is up for discussion [in the app-history repo][initial-event-discuss].
+This is up for discussion [in the navigation-api repo][initial-event-discuss].
```

Comment:
IMO we should remove this paragraph; I think we settled reasonably firmly on not including such an event.